app/collector/data_collection_logging.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime
+import logging
+import os
+
+class DataCollectionLogging():
+# Class to implement the logging for the collection data engine.
+
+    def __init__(self):
+
+        # Config log file
+        today = datetime.today().strftime('%Y-%m-%d')
+        file_path = 'collector_logs'
+        if not os.path.exists(file_path):
+            os.makedirs(file_path)
+        
+        filename='collector_logs/DataCollectionEngine_'+today+'.log'
```

Comment:
The logging package supports rotating the log each day. You don't need to implement it, just configure it.